docs/product/visualizations.md

```diff
@@ -11,141 +11,223 @@ tags: [product]
 Visualizations have two main purposes:
 
 - **Display results of nodes**  
-  Each node can be assigned with one or more visualization. After a node
-  computes its new value, the visualization shows it in an understandable way to
-  the user. Please note that a single node can be assigned with multiple
-  visualizations at the same time. For example, a node might want to display a
-  map of locations and their list at the same time next to each other.
+  Each node can be assigned with one or more visualization. After a node computes its new value, the visualization shows
+  it in an understandable way to the user. Please note that a single node can be assigned with multiple visualizations
+  at the same time. For example, a node might want to display a map of locations, and their list at the same time next
+  to each other.
```

Comment:
This text should be reformatted back. We keep rust sourcecodewith 100 charslimit but everything else, including all docs, with 80 chars limit.